lib/pure/stats.nim

```diff
@@ -53,6 +53,37 @@ runnableExamples:
   doAssert statistics.kurtosis() ~= -1.0
   doAssert statistics.kurtosisS() ~= -0.7000000000000008
 
+  var 
+    ASv = [7,4,6,3,9,1]        #  sorted   1 3 4 6 7 9
+    A   : array[len(ASv),int]
+
+  func myCmp(x,y:int):int =
+    if x==y : 0 elif x<y : -1 else: 1
```

Comment:
the only other option is to put it in std/algorithm instead of std/stats; let's think a bit more

Reply:
The proc ``quickSelect`` is a relative of the ``quicksort`` algorithm. So, ``quickSelect`` could be a part of std/algorithm.
But, ``median`` is a well-known mathematical function from statistics.